website/app/controllers/show.js

```diff
@@ -7,10 +7,27 @@ import { A } from '@ember/array';
 import { showdownConfig } from '../shared/showdown-config';
 
 export default class ShowController extends Controller {
+  queryParams = [
+    {
+      selectedTab: 'tab',
+    },
+  ];
+
+  selectedTab = 'guidelines';
```

Comment:
TODO: this isn't the right default and speaks to maybe turning this into an integer index value

Reply:
Because we don't know the tabs until after the page is rendered we can't change this based on which page we're on. for now I'll delete this which means every tab will have a query param, even the default.